spec/grape/endpoint_spec.rb

```diff
@@ -420,6 +420,22 @@ def app
         expect(last_response.status).to eq(201)
         expect(last_response.body).to eq('Bob')
       end
+
+      it 'returns a 400 if given an invalid multipart body' do
+        # Rack swallowed this error until v2.2.0
+        major, minor, _patch = Rack.release.split('.').map(&:to_i)
```

Comment:
The "correct" way to do this is to use `Gem::Version.new(Rack.release) >= ...`, see https://code.dblock.org/2020/07/16/comparing-version-numbers-in-ruby.html if it's helpful. 

Let's also externalize the `if` as a spec condition so we see it properly skipped.

Reply:
Thanks!  I figured there had to be a better way.